scripts/ci/credscan/CredScanSuppressions.json

```diff
@@ -171,7 +171,8 @@
         "src\\azure-cli\\azure\\cli\\command_modules\\vm\\tests\\latest\\recordings\\test_vm_managed_disk.yaml",
         "src\\azure-cli\\azure\\cli\\command_modules\\vm\\tests\\latest\\recordings\\test_vm_update_with_workspace_linux.yaml",
         "src\\azure-cli\\azure\\cli\\command_modules\\vm\\tests\\latest\\recordings\\test_vm_update_with_workspace_windows.yaml",
-        "src\\azure-cli\\azure\\cli\\command_modules\\vm\\tests\\latest\\recordings\\test_vm_boot_diagnostics.yaml"
+        "src\\azure-cli\\azure\\cli\\command_modules\\vm\\tests\\latest\\recordings\\test_vm_boot_diagnostics.yaml",
+        "src\\azure-cli\\azure\\cli\\command_modules\\vm\\tests\\latest\\recordings\\test_snapshot_access.yaml"
       ],
       "_justification": "[VM] response body contains random value recognized as secret in outdated recoding files of vm may remove in the future, and one-off password"
```

Comment:
`test_snapshot_access` is not a VM related test. Please create a new item to exclude snapshot related test